### ISSUE TITLE
feature request: rt-template with in-place child component (not a function)

### ISSUE THREAD

**Issue by yatra9** [wix/react-templates#214]
Hello.

It would be nice if `rt-template` has a option to put a React component itself (not a function) as a child property.

I use Microsoft's [WinJS](http://try.buildwinjs.com/) toolkit through [react-winjs](https://github.com/winjs/react-winjs).
Some react-winjs components require other renderable React components as their properties.

For example, the `ReactWinJS.SplitView` of [this sample code](https://github.com/winjs/react-winjs/blob/master/examples/showcase/examples/SplitView.jsx) has child properties `paneComponent` and `contentComponent` which are also React components.

For example, `rt-template` with `in-place` property (instead of `arguments` property) makes sense?
```
<ReactWinJS.SplitView
  id={splitViewId}
  <rt-template prop="paneComponent" in-place />
     <div>...</div>
  </rt-template>
  <rt-template prop="contentComponent" in-place />
     <h2 className="win-h2" >{this.state.content}</h2>
  </rt-template>
</ReactWinJS.SplitView>
```

**Comment by nippur72** [comment]
Yes, this is something we really need if we want to talk with other libraries (WinJs but also MaterialUI, Antd etc). We discussed it in various places, see #209, #103, #150 and #155.

**Comment by yatra9** [comment]
Sorry for duplicate issue. I was not aware of those.

By the way, the tag name `rt-template` is a little bit weird, I think.
`rt-inject` or `rt-modify` etc. would be fit better?
